ivy_tests/test_ivy/test_functional/test_nn/test_activations.py

```diff
@@ -12,7 +12,7 @@
 @handle_test(
     fn_tree="functional.ivy.relu",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
```

Comment:
`numeric` already includes `float_and_complex`, so I think you can just leave it as that